pkg/ui/src/views/cluster/containers/nodeGraphs/index.tsx

```diff
@@ -48,7 +49,8 @@ const dashboards: {[key: string]: GraphDashboard} = {
   "replication": { label: "Replication", component: replicationDashboard },
   "distributed": { label: "Distributed", component: distributedDashboard },
   "queues": { label: "Queues", component: queuesDashboard },
-  "requests": { label: "Slow Requests", component: requestsDashboard},
+  "requests": { label: "Slow Requests", component: requestsDashboard },
+  "hardware": { label: "Hardware", component: hardwareDashboard },
```

Comment:
I would sort it much higher, maybe second after "overview"? @piyush-singh @josueeee @Amruta-Ranade thoughts?

Reply:
Done